FhirToDataLake/azurepipelines/FhirToSynapseTest/synapse-e2e-test.yml

```diff
@@ -156,13 +169,19 @@ steps:
       $storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName $(resourceGroup) -AccountName $(storageAccountName)).Value[0]
       $storageContext = New-AzStorageContext -StorageAccountName $(storageAccountName) -StorageAccountKey $storageAccountKey
       Remove-AzStorageContainer -Name $(containerName) -Context $storageContext -Force
+      Remove-AzStorageTable -Name $(containerName)jobinfotable -Context $storageContext -Force
+      Remove-AzStorageTable -Name $(containerName)metadatatable -Context $storageContext -Force
+      Remove-AzStorageQueue -Name $(containerName)jobinfoqueue -Context $storageContext -Force
```

Comment:
If all release pipelines share same table/queue, the remove operation cause concurrent issue when two releases are going on in the same time. But I feel this should not be a big problem in reality since release are triggered not so frequently?

Reply:
The **$(containerName)** here will be different among pipeline buildings, but the conflict does happen when multiple releases try to re-deploy the same test Container Instance with different build images, in that case the latter one will success if it not be affected by another.